guides/source/form_helpers.md

```diff
@@ -838,25 +965,29 @@ Hashes can be nested as many levels as required, for example:
 <input id="person_address_city" name="person[address][city]" type="text" value="New York"/>
 ```
 
-will result in the `params` hash being
+The above will result in the `params` hash being
 
 ```ruby
 { 'person' => { 'address' => { 'city' => 'New York' } } }
 ```
 
-Normally Rails ignores duplicate parameter names. If the parameter name ends with an empty set of square brackets `[]` then they will be accumulated in an array. If you wanted users to be able to input multiple phone numbers, you could place this in the form:
+The other structure is an Array. Normally Rails ignores duplicate parameter names, but if the parameter name ends with an empty set of square brackets `[]` then the parameters will be accumulated in an Array.
+
+For example, if you want users to be able to input multiple phone numbers, you could place this in the form:
 
 ```html
 <input name="person[phone_number][]" type="text"/>
 <input name="person[phone_number][]" type="text"/>
 <input name="person[phone_number][]" type="text"/>
 ```
 
-This would result in `params[:person][:phone_number]` being an array containing the inputted phone numbers.
+This would result in `params[:person][:phone_number]` being an array containing the submitted phone numbers.
```

Comment:
For consistency it would be nice to show the result here as well:
````suggestion
This would result in `params[:person][:phone_number]` being an array containing the submitted phone numbers.

```ruby
{ 'person' => { 'phone_number' => ['555-0123', '555-0124', '555-0125'] } }
```
````
```

Reply:
Updated.